_rules/descriptive-link-5effbb.md

```diff
@@ -0,0 +1,270 @@
+---
+id: 5effbb
+name: Link in context is descriptive
+rule_type: atomic
+description: | 
+ This rule checks that all links in their context describe their purpose.
+accessibility_requirements: 
+  wcag20:2.4.4: # Link Purpose (In Context) 
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Language
+authors:
+  - Carlos Duarte
+  - Marie Trudelle 
+  - Ramit Garg
+---
+
+## Applicability
+
+This rule applies to any HTML or SVG element with the [semantic role](#semantic-role) of [`link`](https://www.w3.org/TR/wai-aria/#link) or a role that inherits from the `link` role, that is [included in the accessibility tree](#included-in-the-accessibility-tree) and has an [accessible-name](#accessible-name) that does not only consist of [whitespace](#whitespace).
+
+## Expectation
```

Comment:
Consider adding a note to say something about what is sufficient description of the purpose of the link. Such as: "Examples of sufficient description would be the name of the linked document or web application. However, this is not an absolute requirement, as other things may also describe the purpose of the link."